ksqldb-common/src/main/java/io/confluent/ksql/util/KsqlConstants.java

```diff
@@ -44,8 +44,8 @@ private KsqlConstants() {
 
   public static final String KSQL_SERVICE_ID_METRICS_TAG = "ksql_service_id";
   public static final String KSQL_PULL_QUERY_SOURCE_TAG = "ksql_pull_query_source";
-  public static final String KSQL_PULL_QUERY_PLAN_TYPE_TAG = "ksql_pull_query_plan_type";
-  public static final String KSQL_PULL_QUERY_ROUTING_TYPE_TAG = "ksql_pull_query_routing_type";
+  public static final String KSQL_PULL_QUERY_PLAN_TYPE_TAG = "query_plan_type";
+  public static final String KSQL_PULL_QUERY_ROUTING_TYPE_TAG = "query_routing_type";
```

Comment:
is there any backwards compatibility concerns here or have these not been published yet?

Reply:
They've only been published on devel, and I don't believe that's a problem.  The metrics pipeline ignores metrics with tags it doesn't recognize.

Reply:
Are we staking a claim to the word "query" to mean "pull query"? I'm wondering if "pull_routing_type" is more identifiable.